angularjs-portal-home/src/main/webapp/js/app-config.js

```diff
@@ -26,7 +26,8 @@ define(['angular'], function(angular) {
         .constant('NAMES', {
             'title' : 'MyUW',
             'crest' : 'img/uwcrest_web_sm.png',
-            'crestalt' : 'UW Crest'
+            'crestalt' : 'UW Crest',
+            'sublogo' : ''
```

Comment:
My bad; design had changed and I'd forgotten the conversation.  Dropping the Beta sub-logo works.